internal/daysteps/daysteps.go

```diff
@@ -12,9 +22,49 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	splitData := strings.Split(data, ",")
+	if len(splitData) != 2 {
+		return 0, 0, fmt.Errorf("%w", ErrIncorrectFormat)
```

Comment:
лучше поподробнее описывать ошибку
```suggestion
		return 0, 0, fmt.Errorf("length of data is not 2")
```